src/main/java/org/parabot/core/asm/wrappers/Callback.java

```diff
@@ -49,4 +49,19 @@ public AddCallbackAdapter getAdapter() {
                 this.invokeMethod, this.desc, this.args, this.conditional);
     }
 
+    @Override
+    public String toString() {
```

Comment:
String.format would be better here too

Reply:
Same thing here, have to perform a null check on the method field.

Reply:
Yep, no worries about this one